string      robot_name
string      planner
float64[]   start
float64[]   goal
float64[]   path
duration    execution_time 
